anime_downloader/commands/ezdl.py

```diff
@@ -0,0 +1,198 @@
+import logging
+import os
+
+import click
+import requests_cache
+
+from anime_downloader import session, util
+from anime_downloader.__version__ import __version__
+from anime_downloader.sites import get_anime_class, ALL_ANIME_SITES
+from anime_downloader import animeinfo
+from anime_downloader.config import Config
+
+logger = logging.getLogger(__name__)
+
+echo = click.echo
+sitenames = [v[1] for v in ALL_ANIME_SITES]
+
+# NOTE: Don't put defaults here. Add them to the dict in config
+@click.command()
+@click.argument('anime_url')
+@click.option(
+    '--episodes', '-e', 'episode_range', metavar='<int>:<int>',
+    help="Range of anime you want to download in the form <start>:<end>")
+@click.option(
+    '--play', 'player', metavar='PLAYER',
+    help="Streams in the specified player")
+@click.option(
+    '--quality', '-q', type=click.Choice(['360p', '480p', '720p', '1080p']),
+    help='Specify the quality of episode. Default-720p')
+@click.option(
+    '--force-download', '-f', is_flag=True,
+    help='Force downloads even if file exists')
+@click.option(
```

Comment:
Is file format required?

Reply:
No, I always use config for that since it's too long. Removing it.